diff --git a/node_modules/sveltestrap/src/Popover.svelte b/node_modules/sveltestrap/src/Popover.svelte
index dc1a5b2..941591d 100644
--- a/node_modules/sveltestrap/src/Popover.svelte
+++ b/node_modules/sveltestrap/src/Popover.svelte
@@ -1,6 +1,6 @@
 <script>
   import { onMount } from 'svelte';
-  import { createPopper } from '@popperjs/core/dist/esm/popper';
+  import { createPopper } from '@popperjs/core';
   import classnames from './utils';
   import InlineContainer from './InlineContainer.svelte';
   import Portal from './Portal.svelte';
diff --git a/node_modules/sveltestrap/src/Tooltip.svelte b/node_modules/sveltestrap/src/Tooltip.svelte
index d7fb9ee..df45967 100644
--- a/node_modules/sveltestrap/src/Tooltip.svelte
+++ b/node_modules/sveltestrap/src/Tooltip.svelte
@@ -1,6 +1,6 @@
 <script>
   import { onDestroy, onMount } from 'svelte';
-  import { createPopper } from '@popperjs/core/dist/esm/popper';
+  import { createPopper } from '@popperjs/core';
   import classnames, { uuid } from './utils';
   import InlineContainer from './InlineContainer.svelte';
   import Portal from './Portal.svelte';
diff --git a/node_modules/sveltestrap/src/popper.js b/node_modules/sveltestrap/src/popper.js
index 9f5f4ea..223675e 100644
--- a/node_modules/sveltestrap/src/popper.js
+++ b/node_modules/sveltestrap/src/popper.js
@@ -1,4 +1,4 @@
-import { createPopper } from '@popperjs/core/dist/esm/popper';
+import { createPopper } from '@popperjs/core';
 
 // Code derived from https://github.com/bryanmylee/svelte-popperjs/blob/master/src/index.ts
 export function createPopperActions(initOptions) {
